smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -162,6 +162,11 @@ public Task Install()
                 commandName = "apt-get";
                 commandArgs = $"--yes install {PackageName}";
             }
+            else if (this.archivePath.Contains(".rpm") && !this.archivePath.Contains(".dep"))
+            {
+                commandName = "sudo";
+                commandArgs = $"rpm --force -U {this.archivePath}";
```

Comment:
This command shouldn't require sudo...the tests themselves run as sudo. Notice the commands in the other branches of this "if" block (apt-get and dpkg) don't require sudo...

Reply:
Changed command name to be rpm instead of sudo